README.md

```diff
@@ -23,12 +22,14 @@ A cli to browse and watch anime. This tool scrapes the site [gogoanime](https://
 ## Usage
 
   ```text
-    ani-cli [-v | -i] [-q <quality>] [-d | -p <download_dir>] [<query>]
-    ani-cli [-v | -i] [-q <quality>] -c
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] [-d | -p <download_dir>] [<query>]
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] -c
     ani-cli -h | -D | -U | -V
+    ani-cli -e [<arguments>]
```

Comment:
remove that line

Reply:
Oops. I meant to delete it but then only did it for the script.